components/dashboard/cards/testimonial.tsx

```diff
@@ -0,0 +1,67 @@
+import { useDispatch } from 'react-redux'
+import { AppDispatch } from '@/lib/redux/store'
+import { setTestimonials, setDisplay } from '@/lib/redux/slices/testimonials'
+import { StyledTestimonialCard, StyledTestimonialInput } from './StyledCards'
+import { Close } from '@mui/icons-material'
+import { Button } from '@mui/material'
+import { useState } from 'react'
+import { createTestimonials } from '@/lib/db/cms/testimonials'
+
+const TestimonialCard = ({ field }: any) => {
+    const dispatch = useDispatch<AppDispatch>()
+    const [isFlipped, setIsFlipped] = useState(false)
+    const { display, email, id, name, rating, review } = field
+
+    const handleDisplayChange = (event: {
+        target: { id: string; checked: boolean }
+    }) => {
+        const { id, checked } = event.target
```

Comment:
I am so confused. There is no way this is the right way to write the types. I think you wanted to do this:
```
  const handleDisplayChange = (event: FormEvent<HTMLInputElement>) => {
        const { id, checked } = event.target
```